app/graphql/types/book_type.rb

```diff
@@ -15,8 +15,12 @@ class BookType < Types::BaseObject
     field :borrower, Types::UserType, null: false
 
     def borrower
-      borrower_id = UserBook.where(user_id: object.user_id).where(book_id: object.id).first.borrower_id
-      User.find(borrower_id)
+      borrower_id = UserBook.where(book_id: object.id).first.borrower_id
```

Comment:
Is using `.first`going to guarantee the most recent borrower (assuming that's what the purpose of this method is)?

Reply:
I'm realizing now I might misunderstand the way you're using this join table. Will there be functionality to "return" a book that would erase this relationship between a user and a book? Or will this field be rewritten every time someone borrows it?

Reply:
After your comment, I realized the way this is written is confusing. `.where` returns an array of elements so by using `.first` I was extracting the element from the array being returned. In this case, a book can only have one borrower so the array will always return just one element. I am going to rewrite this to use the `.find_by` method instead of `.where` so it returns just the element (and not the element in an array).